lib/restaurant.rb

```diff
@@ -0,0 +1,49 @@
+#require 'terminal-table'
+require 'twilio-ruby'
+require './lib/sms.rb'
+
+class Restaurant
+
+  def initialize #smsclass <= suggested by peer
```

Comment:
Yes, very good idea to take an argument to `initialize`. That way you can use any kind of `SmsSender` and it'll 'just work' (i.e. not just Twilio)